src/controllers/hid/hidcontroller.h

```diff
@@ -89,7 +89,10 @@ class HidController final : public Controller {
     hid_device* m_pHidDevice;
     HidControllerPreset m_preset;
 
-    unsigned char m_pPollData[255];
+    static constexpr int m_iNumBuffers = 2;
```

Comment:
We prefix constants with k. We have no i prefix.

Reply:
I am unsure, but I think we should also omit the m_ because it is no actual member variable it only shares the class namespace.

Reply:
```suggestion
    static constexpr int kNumBuffers = 2;
```